.github/bin/ght-render.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+set -euf
+
+GHT_CONF=.github/ght.yaml
+
+render()
+{
+    in=$1
+    out=$2
+    ght_conf=${3:-$GHT_CONF}
+
+    jinja2 \
+        --format yaml\
+        -e jinja2_time.TimeExtension \
+        -o $out \
+        $in $ght_conf
+}
+
+converged()
+{
+    diff -q $1 $2 > /dev/null
+}
+
+tree_converged()
+{
+    templ=$1;
+    rendered=$2;
+
+    # List all the types under the repo, ignoring the .git dir
+    find . -path ./.git -prune -o ${@:3}  | grep -v '^\./\.git' | sort > ${templ}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/manual/sodre/edag-cli/pullRequest?prid=4748186)